contracts/rewards/RewardsController.sol

```diff
@@ -68,7 +71,7 @@ contract RewardsController is RewardsDistributor, VersionedInitializable, IRewar
   }
 
   /// @inheritdoc IRewardsController
-  function configureAssets(RewardsDistributorTypes.RewardsConfigInput[] memory config)
+  function configureAssets(DataTypes.RewardsConfigInput[] memory config)
     external
     override
     onlyEmissionManager
```

Comment:
Think we can skip fetching the totalSupply and and skip the call of _updateRewardData in _configureAssets (just set to 0 reward.lastUpdateTimestamp).